llm-models/embedding/bge/bge-large/01_load_inference.py

```diff
@@ -0,0 +1,101 @@
+print("Cosine-Similarity:", cos_sim)
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC For s2p(short query to long passage) retrieval task, each short query should start with an instruction `Represent this sentence for searching relevant passages:`.
+
+# COMMAND ----------
+
+queries = ["What type of organism is commonly used in preparation of foods such as cheese and yogurt?"]
+passages = [
+  "Mesophiles grow best in moderate temperature, typically between 25°C and 40°C (77°F and 104°F). Mesophiles are often found living in or on the bodies of humans or other animals. The optimal growth temperature of many pathogenic mesophiles is 37°C (98°F), the normal human body temperature. Mesophilic organisms have important uses in food preparation, including cheese, yogurt, beer and wine.", 
+  "Without Coriolis Effect the global winds would blow north to south or south to north. But Coriolis makes them blow northeast to southwest or the reverse in the Northern Hemisphere. The winds blow northwest to southeast or the reverse in the southern hemisphere.",
+  "Summary Changes of state are examples of phase changes, or phase transitions. All phase changes are accompanied by changes in the energy of a system. Changes from a more-ordered state to a less-ordered state (such as a liquid to a gas) areendothermic. Changes from a less-ordered state to a more-ordered state (such as a liquid to a solid) are always exothermic. The conversion of a solid to a liquid is called fusion (or melting). The energy required to melt 1 mol of a substance is its enthalpy of fusion (ΔHfus). The energy change required to vaporize 1 mol of a substance is the enthalpy of vaporization (ΔHvap). The direct conversion of a solid to a gas is sublimation. The amount of energy needed to sublime 1 mol of a substance is its enthalpy of sublimation (ΔHsub) and is the sum of the enthalpies of fusion and vaporization. Plots of the temperature of a substance versus heat added or versus heating time at a constant rate of heating are calledheating curves. Heating curves relate temperature changes to phase transitions. A superheated liquid, a liquid at a temperature and pressure at which it should be a gas, is not stable. A cooling curve is not exactly the reverse of the heating curve because many liquids do not freeze at the expected temperature. Instead, they form a supercooled liquid, a metastable liquid phase that exists below the normal melting point. Supercooled liquids usually crystallize on standing, or adding a seed crystal of the same or another substance can induce crystallization."
+  ]
+instruction = "Represent this sentence for searching relevant passages: "
+query_with_instruction = [instruction+q for q in queries]
+q_embeddings = model.encode(query_with_instruction, normalize_embeddings=True)
+p_embeddings = model.encode(passages, normalize_embeddings=True)
+
+scores = util.cos_sim(q_embeddings, p_embeddings)
+print("Cosine-Similarity scores:", cos_sim)
```

Comment:
Should print `scores` instead.
```suggestion
print("Cosine-Similarity scores:", scores)
```